src/shared/thunks/app.ts

```diff
@@ -0,0 +1,31 @@
+import {Dispatch} from 'redux'
+import {getAPIBasepath} from 'src/utils/basepath'
+import {VersionInfo} from 'src/types'
+import {Action, setVersionInfo} from 'src/shared/actions/app'
+
+export const fetchVersionInfo = () => async (
+  dispatch: Dispatch<Action>
+): Promise<VersionInfo> => {
+  try {
+    const url = `${getAPIBasepath()}/health`
+    const response = await fetch(url)
```

Comment:
Have to use `fetch` here since the oats client doesn't work with the `servers` option in the oss swagger: https://github.com/influxdata/openapi/blob/35d734671d05ba9337dd6fbd8bb5c6085482011b/contracts/oss.yml#L5887

Reply:
:+1: can you add that comment to the code please?